Gemfile

```diff
@@ -141,6 +141,7 @@ group :production, :staging do
   gem 'unicorn-worker-killer'
 end
 
+
```

Comment:
Why did you add this line?

Reply:
Actually I added a line with content, then deleted the content but without the line itself. Will delete the line.